examples/data-objects/table-document/package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@fluid-example/table-document",
 	"version": "2.0.0-internal.4.3.0",
+	"private": true,
```

Comment:
![image](https://media.giphy.com/media/l3q2K5jinAlChoCLS/giphy.gif)

On an example? Wow haha.

Reply:
Yeah, it's evolved quite a bit over time (originally was less of an "example" and more of a ready-to-use table data storage object), and our offerings for table-style data have improved since then too.

Reply:
Yet another reason why commentless JSON is a terrible format for package config. 😞